grammars/repositories/flavors/math-block.cson

```diff
@@ -1,13 +1,35 @@
 key: 'math-block'
 patterns: [
+  {
+    name: 'block.math.markup.md'
+    begin: '(\\${2})'
+    end: '(\\${2})(?:.*)'
```

Comment:
`(?:.*)` is incorrect. It should be just
```cson
end: '(\\${2})'
```

Pandoc, at least, allows Markdown to be on the same line as the closing `$$`. I.e.:
```md
# testing

$$
a = \sum \frac 1 i
$$ *markdown on same line*
```

is converted into: 
![image](https://user-images.githubusercontent.com/15164633/42195848-fb0513e0-7e48-11e8-89e3-f4f1d5067922.png)

Reply:
Thank you for the suggestion, and sorry for my late reaction. I have just fixed it accordingly.